tests/unit/examples/test_notebooks_pyspark.py

```diff
@@ -28,12 +30,20 @@ def test_data_split_runs(notebooks, output_notebook, kernel_name):
 
 @pytest.mark.notebooks
 @pytest.mark.spark
+@pytest.mark.fake_movielens
 @pytest.mark.skipif(
     sys.platform == "win32", reason="Takes 2764.50s in Windows, while in Linux 124.35s"
 )
-def test_als_deep_dive_runs(notebooks, output_notebook, kernel_name):
+@pytest.mark.parametrize("data_size", ["mock100"])
+def test_als_deep_dive_runs(notebooks, output_notebook, kernel_name, data_size):
     notebook_path = notebooks["als_deep_dive"]
-    pm.execute_notebook(notebook_path, output_notebook, kernel_name=kernel_name)
+    pm.execute_notebook(notebook_path, output_notebook, kernel_name=kernel_name,
+                        parameters=dict(
+                            MOVIELENS_DATA_SIZE=data_size,
```

Comment:
trying to think of simpler solutions, why not just input `MOVIELENS_DATA_SIZE="mock100"` instead of adding `@pytest.mark.fake_movielens` and `@pytest.mark.parametrize("data_size", ["mock100"])`

Reply:
`@pytest.mark.fake_movielens` and the parametrization were artifacts of producing the offline experiment results included in this PR. It's mainly there so I can do something like "pytest tests/unit -m 'fake_movielens'" 😄. If you think the results are convincing enough and we don't need to regenerate them, I can remove them.

Reply:
We are trying not to add too many custom marks, otherwise, we will clutter the repo too much. 
I think in general, whenever we can reduce or simplify the code in the repo, we are facilitating the work of our users